daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -1068,6 +1068,34 @@ class RazerBlackWidowV3MiniHyperspeedWireless(RazerBlackWidowV3MiniHyperspeed):
     USB_PID = 0x0271
 
 
+class RazerBlackWidowV4MiniHyperspeed(_RippleKeyboard):
+    """
+    Class for the Razer BlackWidow V4 Mini Hyperspeed
```

Comment:
Add `(Wired)` to this comment, add `Wired` to class name.

And make sure HyperSpeed is capitalized consistently.